web-client/public/js/setup-load-and-import-handlers.js

```diff
@@ -171,8 +171,9 @@ export const setupLoadAndImportHandlers = (grnState) => {
      * for helping to resolve this.
      */
 
-    // $(".upload").change(uploadHandler(loadGrn));
-    $("body").on("change", ".upload", uploadHandler(loadGrn));
+    $("body").on("change", "#upload-network", uploadHandler(loadGrn));
+    $("body").on("change", "#upload-nav", uploadHandler(loadGrn));
```

Comment:
Per the discussion at our meeting, let’s try reverting this change so that we retain having a single line of code to designate this behavior, since it remains the same for both elements
```suggestion
   $("body").on("change", ".upload", uploadHandler(loadGrn));
```